meta-balena-common/recipes-support/os-helpers/os-helpers.bb

```diff
@@ -4,7 +4,7 @@ LICENSE = "Apache-2.0"
 LIC_FILES_CHKSUM = "file://${BALENA_COREBASE}/COPYING.Apache-2.0;md5=89aea4e17d99a7cacdbeed46a0096b10"
 
 DEPENDS = "time-native curl-native"
-RDEPENDS:${PN}-fs = "e2fsprogs-tune2fs mtools parted bash util-linux-fdisk"
+RDEPENDS:${PN}-fs = "e2fsprogs-tune2fs mtools parted bash util-linux-fdisk zstd"
```

Comment:
Just for my understanding, can you remind me why we use zstd in the initrd and lz4 in the final OS?

Reply:
Unfortunately I can't - Joseph did the initial implementation and it was using zstd already, I don't remember if we challenged it. I don't think it's that important tbh.